test/helpers.bash

```diff
@@ -66,23 +80,41 @@ function get_container_netns_path() {
 # it joins the test network namespace before it invokes $NETAVARK,
 # which may be 'netavark' or '/some/path/netavark'.
 function run_netavark() {
-    run_in_host_netns $NETAVARK "$@"
+    local first=$1
+    shift 1
+    run_in_host_netns "$first" $NETAVARK "$@"
 }
 
 ################
 #  run_in_container_netns  #  Run args in container netns
 ################
 #
 function run_in_container_netns() {
-    run_helper nsenter -n -t $CONTAINER_NS_PID "$@"
+    if [ "$1" == "1" ]; then
+        shift 1
+        run_helper nsenter -n -t $CONTAINER1_NS_PID "$@"
+    elif [ "$1" == "2" ]; then
+        shift 1
+        run_helper nsenter -n -t $CONTAINER2_NS_PID "$@"
+    else    
+        die
+    fi
+
 }
 
 ################
-#  run_in_host_netns  #  Run args in host netns
+#   run_in_host_netns  #  Run args in host netns
 ################
-#
 function run_in_host_netns() {
-    run_helper nsenter -n -t $HOST_NS_PID "$@"
+    if [ "$1" == "1" ]; then
+        shift 1
+        run_helper nsenter -n -t $HOST1_NS_PID "$@"
+    elif [ "$1" == "2" ]; then
+        shift 1
+        run_helper nsenter -n -t $HOST2_NS_PID "$@"
+    else 
+        die
+    fi
```

Comment:
The hostns can only be one netns! You cannot have more the one host.

Reply:
Ok, that makes sense. making multiple container namespaces makes sense though, right?